src/Estimators/src/FloatingBaseEstimator.cpp

```diff
@@ -69,14 +69,25 @@ bool FloatingBaseEstimator::initialize(std::weak_ptr<BipedalLocomotion::Paramete
         return false;
     }
 
-    // setup model related entities
-    auto modelHandle = handle->getGroup("ModelInfo");
-    if (!setupModelParams(modelHandle))
+    if (m_useModelInfo)
     {
-        std::cerr << "[FloatingBaseEstimator::initialize] "
-        "Could not load model related parameters."
-        << std::endl;
-        return false;
+        if (!m_modelComp.isKinDynValid())
+        {
+            std::cerr << "[FloatingBaseEstimator::initialize] The kindyn object with valid does not seem to be loaded."
+            << "Please call initialize(handler, kindyncomputations) to set the kindyn object."
```

Comment:
Does it make sense to have a way to set only the kindyn object?

Reply:
This method already exists, https://github.com/dic-iit/bipedal-locomotion-framework/blob/afae4bad37b4a09ed5700b3657fea90c6f6ca40e/src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimator.h#L53 within the nested class `ModelComputations` .

If this is what you mean?

Reply:
Ah ok, great. Does it make sense to add it as a suggestion in the error message?

Reply:
Yep, I will update it.

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/254/commits/10e3009d2a596e5fe5285718b91c776db4683086